### PR TITLE
Create documentation

### PR DESCRIPTION
**Pull request by Emma-MacK**
Starting documentation files for:

- FrontendDocs.md   
- Future_development.md  
- Installation.md
- Running_Docker.md  
- 'Running make bed tests.md' 
-  Seeking_help.md

### CODE REVIEW

File: docs/Installation.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+Or via ssh//
+
+for example:
+
+`git clone https://github.com/Emma-MacK/Software_assessment`
+
+N.B. We ask that third party users create their own copy of the github repository before cloning it locally!
+
+The environment for running the tool can be set up with the following commands
```
Comment (EKPayne) [inline docs/Installation.md:18]: I think this needs to be moved down to command line as this step is not necessary if running from docker.

File: docs/Installation.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+conda env create -n example_environment -f environment.yml
+conda activate example_environment
+pip install -r src/requirements.txt
+```
+To exit the environment you can use
+
+`conda deactivate`
+
+
+## Docker
```
Comment (EKPayne) [inline docs/Installation.md:30]: Would add a mention that it will require docker to be installed and then link to docker installation documentation

Comment (Emma-MacK) [reply docs/Installation.md:30]: Done

File: docs/Installation.md (hunk 1 above)
Comment (Emma-MacK) [reply docs/Installation.md:18]: I'll a link to the docker section if people want to skip this step

Comment (Emma-MacK) [reply docs/Installation.md:20]: Done

**Review by raymondmiles — COMMENTED** [review]
These are v. good documentation instructions. Cannot see the change requested on this side.

File: docs/Installation.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+Or via ssh//
+
+for example:
+
+`git clone https://github.com/Emma-MacK/Software_assessment`
+
+N.B. We ask that third party users create their own copy of the github repository before cloning it locally!
+
+If you plan on using docker to run the tool, skip to the [docker](#docker) section rather than following the environment set up
```
Comment (Emma-MacK) [inline docs/Installation.md:18]: Lets user skip environment set up if they intend to use docker

File: docs/Installation.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+`conda deactivate`
+
+
+## Docker
+
+Alternatively to creating a conda environment, docker could be used for running commands.
+
+This approach requires that docker has been installed. The official process for installing docker can be found
+[here](https://docs.docker.com/engine/install/ubuntu/)
```
Comment (Emma-MacK) [inline docs/Installation.md:37]: Installation instructions for docker

**Comment by Emma-MacK** [comment]
The changes made to resolve the requests are outside the window shown on this forum it seems, so I have added comments to the lines that address the requested changes

> These are v. good documentation instructions. Cannot see the change requested on this side.

**Comment by EKPayne** [comment]
Happy with changes. Merge request accepted

**Pull request merged**